### PR TITLE
Static Analysis and Security Testing adjustments

### PR DESCRIPTION
**Pull request by cvcrckt**
For now I will limit PRs to one section at a time. One item puzzles me in this this section:

> Identification of the security contexts and system calls, which can compromise a system

I *think* this should be:

> Identification of security contexts and system calls which may compromise a system

... but maybe I am misinterpreting it.

### CODE REVIEW

**Comment by cvcrckt** [comment]
.. and sorry, this has my previous commits under #493 included

**Review by TheFoxAtWork — APPROVED** [review]
looks good i'd like another set of eyes on this to look it over.

**Comment by TheFoxAtWork** [comment]
@PushkarJ can u take alook at this?

File: security-whitepaper/cloud-native-security-whitepaper.md
Comment (RonVider) [inline security-whitepaper/cloud-native-security-whitepaper.md:310]: It’s important to note that static analysis of software code will examine only that specific component (microservice for example) without any context of the surrounding components. Therefore, the ability of such a tool to provide accurate identification of vulnerabilities across the application is very limited. Unfortunately, such a static analysis will result in a high rate FN and FP in a cloud-native environment (if we want to be more specific - in a microservices architecture)

Comment (TheFoxAtWork) [reply security-whitepaper/cloud-native-security-whitepaper.md:310]: @RonVider  i think the distinction is important enough to call it out.  made update
```suggestion
Static analysis of IaC, application manifests, and software code provide linting, identification of misconfigurations, and vulnerabilities of the specific component and will not include context of the surrounding components in analysis.  It is important to note that individual analysis is important, but should not be the only form of analysis. IaC code should be subject to the same pipeline policy controls as are application workloads.
```

File: security-whitepaper/cloud-native-security-whitepaper.md
Comment (lumjjb) [inline security-whitepaper/cloud-native-security-whitepaper.md:310]: > It is important to note that individual analysis is important, but should not be the only form of analysis.

I think that this is a bit too much of a detail for the whitepaper. This is more a consideration for the implementation of the tools rather than the operator implementing these tooling. 

> IaC code should be subject to the same pipeline policy controls as are application workloads.

I would prefer the term similar since it is not going to be exactly the same as workloads.

File: security-whitepaper/cloud-native-security-whitepaper.md
Comment (lumjjb) [inline security-whitepaper/cloud-native-security-whitepaper.md:312]:
```suggestion
IaC is an increasingly popular way for organizations to deploy cloud and container infrastructure. Insecure configurations in IaC templates will naturally lead to vulnerabilities in the deployed infrastructure. These templates should therefore be scanned for characteristics that compromise security, before deploying the application and infrastructure artifacts. Key misconfigurations to keep an eye out for include:
```

Already mentioned earlier about making it a pipeline. I think the idea of having a secure process here is more of the focus than the automation of it.

**Review by lumjjb — COMMENTED** [review]
Couple comments , but otherwise looks good

File: security-whitepaper/cloud-native-security-whitepaper.md
Comment (TheFoxAtWork) [reply security-whitepaper/cloud-native-security-whitepaper.md:314]:
```suggestion
Static analysis of IaC, application manifests, and software code provide linting, identification of misconfigurations, and vulnerabilities of the specific component and will not include context of the surrounding components in analysis.  It is important to note that individual analysis is important, but should not be the only form of analysis. IaC code should be subject to the similar pipeline policy controls as are application workloads.
```

**Comment by lumjjb** [comment]
Hi @cvcrckt, since there are no additional comments, the content looks good. Can you help fix some of the CI checks and we should be good to merge!

**Comment by PushkarJ** [comment]
LGTM to me too, sorry for the delayed review. 

@lumjjb it sounds like the `lint`, `spelling` and `links` checks are unrelated to the updates here. Should we maybe merge this and track fixing of these changes for whitepaper markdown in a separate PR/issue. I can open one up, if it helps :)

**Comment by lumjjb** [comment]
bumping this up, someone with write access can probably help clean this up and then we can merge it

**Review by TheFoxAtWork — APPROVED** [review]
Fixed all the things.

**Comment by PushkarJ** [comment]
Thank you Emily!! Woohoo

**Pull request merged**
